src/async_fuse/memfs/dist/server.rs

```diff
@@ -372,9 +372,9 @@ async fn remove<S: S3BackEnd + Send + Sync + 'static>(
 /// Handle `GetInodeNum` request
 async fn get_inode_num<S: S3BackEnd + Send + Sync + 'static>(
     stream: &mut TcpStream,
-    meta: &Arc<S3MetaData<S>>,
+    _meta: &Arc<S3MetaData<S>>,
 ) -> anyhow::Result<()> {
-    let inum = meta.cur_inum();
-    tcp::write_u32(stream, inum).await?;
+    // let inum = meta.cur_inum();
+    tcp::write_u32(stream, 0).await?;
```

Comment:
Why should we return 0 while handle this request?

Reply:
This part is about old logic of inum allocation, I will remove them.